In [1]:
# Hypothesis Testing for Weekly Operating Cost

import math
from scipy.stats import norm

# -----------------------------
# Given Data
# -----------------------------

sample_mean = 3050        # x̄
X = 600                   # Mean units produced
intercept = 1000
cost_per_unit = 5

# Theoretical Mean
population_mean = intercept + cost_per_unit * X

sigma = 5 * 25            # Population standard deviation
n = 25                    # Sample size
alpha = 0.05

# -----------------------------
# Step 1: State Hypotheses
# -----------------------------

print("Hypothesis Testing")
print("----------------------------")
print("H0: μ <= 4000")
print("H1: μ > 4000")

# -----------------------------
# Step 2: Calculate Test Statistic
# -----------------------------

z = (sample_mean - population_mean) / (sigma / math.sqrt(n))

print("\nTheoretical Mean =", population_mean)
print("Sample Mean =", sample_mean)
print("Population Standard Deviation =", sigma)
print("Sample Size =", n)

print("\nTest Statistic (Z) =", round(z,4))

# -----------------------------
# Step 3: Critical Value
# -----------------------------

critical_value = norm.ppf(1 - alpha)

print("\nSignificance Level =", alpha)
print("Critical Value =", round(critical_value,4))

# -----------------------------
# Step 4: Decision
# -----------------------------

print("\nDecision")

if z > critical_value:
    print("Reject the Null Hypothesis (H0)")
else:
    print("Fail to Reject the Null Hypothesis (H0)")

# -----------------------------
# Step 5: Conclusion
# -----------------------------

print("\nConclusion")

if z > critical_value:
    print("There is sufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
else:
    print("There is insufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")

Hypothesis Testing
----------------------------
H0: μ <= 4000
H1: μ > 4000

Theoretical Mean = 4000
Sample Mean = 3050
Population Standard Deviation = 125
Sample Size = 25

Test Statistic (Z) = -38.0

Significance Level = 0.05
Critical Value = 1.6449

Decision
Fail to Reject the Null Hypothesis (H0)

Conclusion
There is insufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


The hypothesis test was conducted to determine whether the average weekly operating cost of Bombay Hospitality Ltd. franchises is higher than the theoretical cost predicted by the model (W = 1000 + 5X). Based on the given data, the theoretical mean weekly operating cost is **Rs. 4000**, while the sample mean is **Rs. 3050**. The calculated **Z-test statistic is -38.0**, and the critical value at a **5% significance level** is **1.6449**. Since the calculated Z-value is much smaller than the critical value, the null hypothesis is **not rejected**. Therefore, there is **insufficient evidence** to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests. In fact, the sample mean is considerably lower than the theoretical mean, indicating that the observed data does not support the claim of increased operating costs.
